cmd/cli/root.go

```diff
@@ -75,6 +80,9 @@ func getKubernetesClient(kubePath string) (kubernetes.Interface, error) {
 		}
 	}
 
+	config.Impersonate.UserName = options.AsUser
+	config.Impersonate.Groups = options.AsGroups
+
```

Comment:
Can you elaborate more on the use-case for being able to impersonate in the case of modifying aws-auth?

Reply:
this allows the ability to not have to hardcode `--as` and `--as-group` values in the kube config.